src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -31,8 +41,10 @@
             Arrays.asList("collated**", "*.aa1", "**.aa2", "**.java");
     private static final List<String> SECOND_AUTHOR_GLOB_LIST = Arrays.asList("collated**", "**[!(.md)]");
 
-    @Test
-    public void configJson_overridesRepoConfig_success() throws InvalidLocationException, GitDownloaderException {
+    private static RepoConfiguration REPO_DELTA_STANDLONE_CONFIG;
```

Comment:
make it static final, can just initialize here.

Reply:
As discussed, this nit will be ignored. 

There is no direct way to make it a `static final`, solutions may decrease code quality.